e2etest/pkg/executor/executor.go

```diff
@@ -17,16 +17,20 @@ func (hostExecutor) Exec(cmd string, args ...string) (string, error) {
 	return string(out), err
 }
 
-func ForContainer(containerName string) Executor {
-	return &containerExecutor{container: containerName}
+func ForContainer(containerName string, containerRuntime string) Executor {
```

Comment:
I'd rather have a higher level object that returns Executors for a given runtime and having one global of these, or consuming the global containerRuntime internally here.
I don't think the runtime should be part of this since it will always be one on the same e2e test run.